package.json

```diff
@@ -18,6 +18,7 @@
     "pree2e": "npm run update-webdriver",
     "e2e": "gulp e2e:run",
     "e2e-on-target": "gulp e2e:tests",
+    "pree2e-on-target": "npm run update-webdriver",
```

Comment:
This looks like a double up on pree2e. Not a blocker, but think we should remove it next rev.